### ISSUE TITLE
[BUG] - Incorrect Boilerplate in American-British Translator

### ISSUE THREAD

**Issue by nhcarrigan** [freeCodeCamp/freeCodeCamp#39631]
**Describe your problem and how to reproduce it:**

I was looking in to writing tests for the American British Translator project. Poking around the code in the example project, I see these routes in the test runner: 

```js
  app.route('/_api/routes/api.js')
    .get(function(req, res, next) {
      console.log('requested');
      fs.readFile(__dirname + '/routes/api.js', function(err, data) {
        if(err) return next(err);
        res.type('txt').send(data.toString());
      });
    });
  app.route('/_api/controllers/convertHandler.js')
    .get(function(req, res, next) {
      console.log('requested');
      fs.readFile(__dirname + '/controllers/convertHandler.js', function(err, data) {
        if(err) return next(err);
        res.type('txt').send(data.toString());
      });
    });
```

However, neither of these routes point to existing files. These appear to be the routes for the Metric-Imperial converter. 

Is there another way to write the tests? The logic appears to be handled client-side, so I'm not sure how to approach this without modifying these routes to target the appropriate files. I tried fetching the `translator.js` file directly but got hit with a CORS.

**Add a Link to the page with the problem:**

[Example Project Code](https://repl.it/@freeCodeCamp/American-British-English-Translator#routes/fcctesting.js)
[Live Example Project](https://americanbritish-english-translator--freecodecamp.repl.co/)

**Tell us about your browser and operating system:**
* Browser Name: Chrome
* Browser Version: 85.0.4183.102
* Operating System: Windows 10 V1909

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

**Comment by Sky020** [comment]
@nhcarrigan Does #39822 solve (close) this?

**Comment by Sky020** [comment]
Ignore that (_I am unable to edit the above comment, for some reason_), I just remembered the PR is targeting `/learn`.

**Comment by nhcarrigan** [comment]
I'm glad you pinged me on this though. 

The `api.js` file was added to the boilerplate, but we neglected to clean up the erroneous `convertHandler` route. I'll grab @SaintPeter next time we're free to get that route removed. 😁

**Comment by nhcarrigan** [comment]
This should be closed with the merge of [the boilerplate PR](https://github.com/freeCodeCamp/demo-projects/pull/23) but we will need to add a commit to remove that extra route.

**Comment by nhcarrigan** [comment]
This will be closed when freeCodeCamp/demo-projects#23 lands. 🙂
